.vscode/tasks.json

```diff
@@ -0,0 +1,49 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+
+  "tasks": [
+    {
+      "label": "build-and-copy",
+      "dependsOrder": "sequence",
+      "dependsOn": ["build", "make-plugin-dir", "copy-dll"]
+    },
+    {
+      "label": "build",
+      "command": "dotnet",
+      "type": "shell",
+      "args": [
+        "publish",
+        "${workspaceFolder}/${config:pluginName}.sln",
+        "/property:GenerateFullPaths=true",
+        "/consoleloggerparameters:NoSummary"
+      ],
+      "group": "build",
+      "presentation": {
+        "reveal": "silent"
+      },
+      "problemMatcher": "$msCompile"
+    },
+    {
+        "label": "make-plugin-dir",
+        "type": "shell",
+        "command": "mkdir",
+        "args": [
+           "-Force",
+           "-Path",
```

Comment:
Are those arguments Windows specific? I've never seen them before and using Linux, only `-p` or `--parents` is required to generate all parent directories (as needed).

Reply:
yep, i was stuck on windows, it's a powershell thing